migrations/__init__.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+MONGO_HOST = os.getenv('MONGO_HOST', 'mongomock://localhost')
+DATABASE = 'normal-oj'
```

Comment:
Does this for the pass of CI?

Reply:
Well...no, the production service also uses these settings to connect to MnogoDB (i.e. `MONGO_HOST` env var and fixed string `normal-oj`).

Reply:
Got it